backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Utils/TestDataUtil.cs

```diff
@@ -63,6 +63,29 @@ public static List<ServiceResourceFE> GetExpectedResources(ResourceType resource
             return filteredResources;
         }
 
+        public static List<ServiceResourceFE> GetSingleRightsResources()
+        {
+            List<ServiceResourceFE> resources = new List<ServiceResourceFE>();
+            List<ServiceResourceFE> filteredResources = null;
+
+            string path = GetResourcesPath("resourcesfe");
```

Comment:
so here in TestDataUtil the file path is resourcesfe but seems like ResourceRegistryClientMock goes to the resource files. Why do they go to different places?

Reply:
That's right. `resources` is the list of all reasources and is in many way's a mock of the resource registry. `resourcefe` however is the same but in the format expected by the frontend, so without the different language alternatives.
In fact, the only difference between the two should be that `resources` contains the resources in the ServiceResource format, while `resourcefe` contains them in the ServiceResourceFE format 😊